libbeat/docs/shared-docker.asciidoc

```diff
@@ -112,7 +112,7 @@ using this syntax:
 [source,shell]
 --------------------------------------------
 -E cloud.id=<Cloud ID from Elasticsearch Service> \
--E cloud.auth=elastic:<elastic password>
+-E cloud.auth=username:<elastic password>
```

Comment:
`elastic` is the default user across almost all of our documentation, it is one of the built in users in Elasticsearch: https://www.elastic.co/guide/en/elasticsearch/reference/current/built-in-users.html

I am hesitant to change this for the sake of consistency across the rest of our documentation, but I could see how this would be confusing if you didn't know this.

You could instead add a note on this line that the default user should be changed if you are using something else. There's an example of how to do this a few lines up:

```
--------------------------------------------
docker run \
  --cap-add="AUDIT_CONTROL" \
  --cap-add="AUDIT_READ" \
  {dockerimage} \
  setup -E setup.kibana.host=kibana:5601 \
  -E output.elasticsearch.hosts=["elasticsearch:9200"] <1>
--------------------------------------------

<1> Substitute your Kibana and Elasticsearch hosts and ports.
```

See the use of the `<1>` to create an anchor for a tip and then define it later.